### ISSUE TITLE
Latest version 4.1.1 (?) requires Node RED version 4.0.2

### ISSUE THREAD

**Issue by FireWizard52** [caputomarcos/node-red-contrib-oauth2#42]
Tried to update to version 4.1.1.. Does this really exist? Not according to Github.
Got message, that it cannot load the module as it requires Node RED version 4.0.2.

I'm running version 3.0.2.

Is this wrong or do I have to wait for approx another year?

Regards

**Comment by caputomarcos** [comment]
Thank you for reaching out. I would like to inform you that version 4.0.2 is actually an incorrect version of the node-red-contrib-oauth2 module and should not be installed. The correct version of Node-RED is version 3.0.2, as you mentioned.

Additionally, version 4.1.1 of the node-red-contrib-oauth2 is now available and can be installed on your current version of Node-RED (3.0.2). I apologize for any confusion that may have occurred regarding this matter.
```bash
https://github.com/caputomarcos/node-red-contrib-oauth2/tree/v4.1.1
````
If you have any other questions or need further information, please do not hesitate to contact us again.

Best regards

**Comment by FireWizard52** [comment]
Hi @caputomarcos 

Thanks for your quick respose
I think that that is exactly the problem, as I installed version 4.1.1 from the palette manager in Node RED.
And this gave me that message, so I thought you were a year ahead with that version 4.1.1.

Regards

**Comment by caputomarcos** [comment]
Hi @FireWizard52 

You're very welcome for the quick response! You know what they say, "never trust a package manager on Monday mornings"! I'm glad we were able to figure out the issue with version 4.1.1.

And don't worry, we all make mistakes sometimes. As for me, I have to clean up this mess I made in the packages. Lesson learned: double-check before hitting that install button!

Let me know if you have any other questions or concerns. Have a great day!

Cheers!
